conf/routes

```diff
@@ -82,6 +86,7 @@ GET     /userapi/labels                         @controllers.UserProfileControll
 GET     /userapi/labelCounts/all                @controllers.UserProfileController.getAllLabelCounts
 GET     /contribution/auditCounts               @controllers.UserProfileController.getAuditCounts
 GET     /contribution/auditCounts/all           @controllers.UserProfileController.getAllAuditCounts
+GET     /userapi/completedMissionCounts/all     @controllers.UserProfileController.getAllUserCompletedMissionCounts
```

Comment:
And what is the difference between /userapi and /contribution? I honestly don't know which route this should fall under.